### PR TITLE
Add "Not reviewed by you" filter

### LINKED ISSUE #1345: Add PR review filter "Not reviewed by you"
I often have the issue that some PRs are still pending review by other contributors,
but I want to just see the PRs I haven't reviewed yet.

There is a **"Reviewed by you"** option, but I want the negation of this
image

This can be achieved by prefixing the `reviewed-by:USERNAME` parameter with a `-`,
so e.g. `-reviewed-by:DerTieran`.

I'm happy to implement this functionality if it is wanted.
My idea would be to add the **"Not reviewed by you"** filter item to the above drop down menu.

### PR DESCRIPTION
**Pull request by DerTieran**
This is the first implementation of the "Not reviewed by you" filter mentioned in #1345

I'm not to sure if the implementation is still to rough, also the page to add this would be something like `PRList` but I didn't find that so for now  I use the `isIssueList` function for that.

Fixes #1345

### CODE REVIEW

**Comment by DerTieran** [comment]
@bfred-it Thanks for your adjustments.

Just because I think I didn't mentioned it, the "not reviewed by you" filter will also list the PRs the you created. That's just how the parameter works but it might not be expected.

**Comment by bfred-it** [comment]
I think that’s an easy change, you can add another negative query

**Comment by DerTieran** [comment]
@bfred-it Sorry I have been away for awhile, do I need to do anything before this can be merged?

**Comment by bfred-it** [comment]
Yes, three things:

- an entry in the readme (with retina screenshot)
- a header comment like 8d640d0 (it can just be a line)
- exclude your own PRs from this new filter (i.e. add <code>\`-author:${currentuser}\`</code>)

**Comment by DerTieran** [comment]
@bfred-it I hope I did everything as you intended.
I noticed a bit of an issue with adding `-author:${username}`, when swithcing between the "Reviews" filter the author query doesn't get removed. So the user needs to remove it manually, I'm not sure what's better having to remove it or seeing also your own PRs.

**Comment by bfred-it** [comment]
Ugh yeah that's right. Maybe that change should be undone and just include your own PRs instead, otherwise there's too much work to update all the links.

**Comment by sindresorhus** [comment]
@DerTieran Can you fix the merge conflict?

File: source/features/add-filter-not-reviewed-by-you.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import * as icons from '../libs/icons';
+import {getUsername} from '../libs/utils';
+
+export default function () {
+	const username = getUsername();
+	const reviewedQuery = encodeURIComponent(`reviewed-by:${username}`);
+
+	const reviewedElement = select(`.select-menu-list a[href*="${reviewedQuery}"]`);
```
[the remark this answers is not in the document]
Comment (DerTieran) [reply source/features/add-filter-not-reviewed-by-you.js]: I'm not sure what you mean here?
This is an existing element in the drop down, I#m using it to insert the feature and build de `href`. If I wanted add an CSS class I would still need to find it first.

**Comment by sindresorhus** [comment]
Looking good. Thanks @DerTieran :)

**Pull request merged**